### ISSUE TITLE
Slow transitions on mobile

### ISSUE THREAD

**Issue by rodrimeli** [greensock/GreenSock-JS#245]
Hi there !

First of all I want to appreciate for this wonderful contribution.
I'm having a problem when I want to show the animation on mobile devices they perform with a kind of lag. I took as an example to start my code, this fiddle https://jsfiddle.net/Lzm78pgs/1/ also written by you and the rows don't move fluently. Any idea what to do for make the transitions more fluid ? Again, thanks for your time.

**Comment by jackdoyle** [comment]
Typically whenever you experience lag in an animation, it has nothing to dow with GSAP - it's the rendering layer. GSAP simply changes the numeric values but then the browser has to do the rendering of those pixels on the screen. In this case, you've got a bunch of SVG graphics which are notoriously CPU-intensive to render. The browser has to fabricate every pixel 60 times per second. It's much harder on the CPU because they're vector graphics (compared to raster graphics). 

In other words, GSAP may be using up 0.1% of the CPU whereas the graphics rendering accounts for 99.9% of the load (I'm not being exact - just an example). In that case, even if I could somehow magically make GSAP 10x faster, you'd still see less than a 0.1% improvement overall. 

The key is to focus on easing the graphics rendering load. One technique you could try is to avoid SVG (maybe canvas, maybe regular DOM elements). 

Does that help at all?

**Comment by rodrimeli** [comment]
You're totally right! Thanks for your help, I will replace those SVGs
